locust/runners.py

```diff
@@ -99,6 +101,7 @@ def spawn_locusts(self, spawn_count=None, stop_timeout=None, wait=False):
         else:
             self.num_clients += spawn_count
 
+        logger.info(locust_runner.running_type)
```

Comment:
I suggest to create better logger info message. maybe message like `Test Ramp Type : Normal` will be more readable